x-pack/plugins/screenshotting/server/screenshots/index.ts

```diff
@@ -49,7 +50,7 @@ import { ScreenshotObservableHandler, UrlOrUrlWithContext } from './observable';
 
 export type { ScreenshotObservableResult, UrlOrUrlWithContext } from './observable';
 
-export interface CaptureOptions extends ScreenshotObservableOptions {
+export interface CaptureOptions extends Optional<ScreenshotObservableOptions, 'urls'> {
```

Comment:
This reverts back to the code in main. The `urls` field has to be optional, because it is mutually exclusive with the `expression` field